common/interpreter/wawaka_wasm/CMakeLists.txt

```diff
@@ -44,20 +47,25 @@ if (NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif ()
 
-if (NOT DEFINED WAMR_BUILD_INTERP)
-  # Enable Interpreter by default
-  set (WAMR_BUILD_INTERP 1)
-endif ()
-
-if (NOT DEFINED WAMR_BUILD_AOT)
-  # Disable AOT by default.
+IF (WASM_MODE STREQUAL "AOT")
+  # Enable AOT.
+  set (WAMR_BUILD_AOT 1)
+  # Disable JIT by default.
+  set (WAMR_BUILD_JIT 0)
```

Comment:
Isn't AOT and JIT mutually exclusive?  But then i guess better safe than sorry :-)

Reply:
I agree :) I copy-pasted this from one of the WAMR CMakeLists. Since we are never really going to use JIT mode, I can just have that mode disabled by default for all WASM modes.